Automattic/wp-calypso
# VideoPress: use new async thumbnail generation when selecting a frame

#### Changes proposed in this Pull Request

This PR changes the call to the thumbnail server in order to use the new async thumbnail generation process for VideoPress videos.
Related to Automattic/greenhouse#1088

#### Testing instructions

* On a sandboxed wpcom site with VideoPress enabled
* Apply D73872-code and follow the instructions for sandboxing
* `yarn start`
* Go to your media library (http://calypso.localhost:3000/media/<your_wordpress_site>)
* Upload a video on VideoPress or use an existing one
* Click on "Edit" > "Edit thumbnail"
![Capture d’écran 2022-01-31 à 16 22 29](https://user-images.githubusercontent.com/1420594/151820847-ab6e74b8-ae2d-4d00-b438-d699aef2bda2.png)
* Seek and click "Select frame"
![Capture d’écran 2022-01-31 à 16 23 06](https://user-images.githubusercontent.com/1420594/151820959-68f5837e-e144-4b01-99a3-a0698864b69c.png)
* ✅ A progress bar should appear
* ✅ Once the thumbnail is generated, the modal should close itself and the media item should be reloaded with the new thumbnail
